packages/@react-aria/tabs/docs/useTab.mdx

```diff
@@ -0,0 +1,195 @@
+</TypeContext.Provider>
+
+`useTab` returns props to be spread onto each individual tab:
+
+<TypeContext.Provider value={docs.links}>
+  <InterfaceType properties={docs.links[docs.exports.useTab.return.id].properties} />
+</TypeContext.Provider>
+
+`useTabPanel` returns props to spread onto the container for the tab content:
+
+<TypeContext.Provider value={docs.links}>
+  <InterfaceType properties={docs.links[docs.exports.useTabPanel.return.id].properties} />
+</TypeContext.Provider>
+
+## Example
+
+This example displays a basic list of tabs. The currently selected Tab get a tabindex of 0 while the rest are set to -1 ensuring that the Tab list is only a single tab stop. The selected tab gets a different style so it's obvious which one is currently selected. Our hooks handle the aria properties that associate the tabs with their tab panel for screen readers. In the tab panel itself, we only render the currently selected item.
+
+```tsx example
+import {Item} from '@react-stately/collections';
+import {useFocus} from '@react-aria/interactions';
+import {mergeProps} from '@react-aria/utils';
+
+function Tabs(props) {
+  let state = useTabListState(props);
+  let tablistRef = React.useRef();
+  let tabpanelRef = React.useRef();
+  let {tabListProps} = useTabList(props, state, tablistRef);
+  let {tabPanelProps} = useTabPanel(props, state, tabpanelRef);
+  return (
+    <div>
+      <div {...tabListProps} ref={tablistRef}>
+        <div style={{display: 'flex', borderBottom: '1px solid grey'}}>
+          {[...state.collection].map((item) => (
+            <Tab key={item.key} item={item} state={state} />
+          ))}
+        </div>
+      </div>
+      <div {...tabPanelProps} ref={tabpanelRef} style={{padding: '10px'}}>
+        {state.selectedItem && state.selectedItem.props.children}
+      </div>
+    </div>
+  );
+}
+function Tab({item, state}) {
+  let {key, rendered} = item;
+  let ref = React.useRef();
+  let {tabProps} = useTab({key}, state, ref);
+  let isSelected = state.selectedKey === key;
+  let selectedStyle = {
+    borderBottom: '3px solid orange'
+  };
+  let tabStyle = {
+    padding: '10px',
+    ...(isSelected ? selectedStyle : {})
+  };
+  return (
+    <div {...tabProps} ref={ref} style={tabStyle}>
+      {item.rendered}
+    </div>
+  );
+}
+
+<Tabs aria-label="History of Ancient Rome">
+  <Item key="FoR" title="Founding of Rome">Arma virumque cano, Troiae qui primus ab oris.</Item>
+  <Item key="MaR" title="Monarchy and Republic">Senatus Populusque Romanus.</Item>
+  <Item key="Emp" title="Empire">Alea jacta est.</Item>
+</Tabs>
+```
+
+## With focusable content
+
+When a Tab Panel has focusable content, there is a difference in behavior. When nothing inside is focusable, the entire `role="tabpanel"` is given a `tabIndex=0`, this is so that the content can be navigated to. When there is something focusable inside, such as a textfield, then this is omitted because the textfield can receive the focus.
```

Comment:
Suggestion for flow
```suggestion
When the tab panel doesn't contain focusable content, the entire panel is given a `tabIndex=0` so that the content can be navigated to. When the tab panel contains focusable content, such as a textfield, then the tab index is omitted because the content itself can receive focus.
```